deploy/clowdapp.yaml

```diff
@@ -109,8 +119,23 @@ parameters:
   - description: Determines Clowder deployment
     name: CLOWDER_ENABLED
     value: "true"
-  - description: Determines if Cloud Watch logging is enabled
-    name: ENABLE_CLOUDWATCH_LOGGING
+  - description: AWS CloudWatch logging integration
+    name: CLOUDWATCH_ENABLED
     value: "false"
+  - description: AWS instance prefix adds string to all instance names, leave blank for production
+    name: AWS_INSTANCE_PREFIX
+    value: ""
+  - description: Internal queue type (memory/sqs/postgres).
+    name: WORKER_QUEUE
+    value: "memory"
+  - description: Internal queue concurrency, how many goroutines to run in parallel.
+    name: WORKER_CONCURRENCY
+    value: "100"
+  - description: Internal queue heartbeat interval. How often to update job liveness information.
+    name: WORKER_HEARTBEAT_SEC
+    value: "30"
+  - description: Internal queue beat threshold. Now many times the beat can be updated until job is cancelled.
+    name: WORKER_MAX_BEATS
+    value: "10"
   - name: OPEN_API_FILEPATH
     value: /var/tmp/openapi.json
```

Comment:
No idea what this is for.

Reply:
@amirfefer isn't this something that should have been deleted? I remember that previously you had OpenAPI loading from the filesystem, but you refactored it to use Go embedding. Can you confirm?